README.md

```diff
@@ -1,38 +1,135 @@
 # Conduit Connector File
 
-### General
-![scarf pixel](https://static.scarf.sh/a.png?x-pxid=42ff59b7-f26d-468d-8c8d-eafc530290cc)
-The File plugin is one of [Conduit](https://github.com/ConduitIO/conduit) builtin plugins.
-It provides both source and destination File connectors, allowing for a file to be either
-a source, or a destination in a Conduit pipeline. 
```

Comment:
I didn't meant to remove this paragraph though 🙈 Just the heading, sorry for the misunderstanding.